src/main/java/Deadline.java

```diff
@@ -0,0 +1,13 @@
+public class Deadline extends Task {
+    protected String by;
+
```

Comment:
Perhaps consider changing the name of the String variable to deadline or dueDate.